src/components/ansatte/Ansatte.less

```diff
@@ -1,6 +1,7 @@
 @import '../../../node_modules/nav-frontend-core/less/_variabler';
+@import '../../../src/basic';
 
-@media (max-width: @screen-lg) {
+@media (max-width: @desktopLimitWidth) {
```

Comment:
Vi må konsekvent bruke enten min-width eller max-width, ellers vil vi have situasjoner hvor begge stylings bliver aktivert på breakpointen

Reply:
Trekker fra 1px for å unngå overlapp.